modules/ossm-applying-certificates-to-multi-cluster-topology.adoc

```diff
@@ -0,0 +1,114 @@
++
+[source,terminal]
+----
+$ export CTX_CLUSTER1=$(oc config current-context)
+----
+
+. Create a namespace called `istio-system` by running the following command:
++
+[source,terminal]
+----
+$ oc get ns istio-system --context "${CTX_CLUSTER1}" || oc create namespace istio-system --context "${CTX_CLUSTER1}"
+----
+
+. Configure {istio} to use `network1` as the default network for the pods on the East cluster by running the following command:
++
+[source,terminal]
+----
+oc --context "${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+----
+
+. Create the CA certificate secret for {istio} on the East cluster by running the following command:
++
+[source,terminal]
+----
+oc get secret -n istio-system --context "${CTX_CLUSTER1}" cacerts || oc create secret generic cacerts -n istio-system --context "${CTX_CLUSTER1}" \
+  --from-file=east/ca-cert.pem \
+  --from-file=east/ca-key.pem \
+  --from-file=east/root-cert.pem \
+  --from-file=east/cert-chain.pem
+----
++
+[NOTE]
+====
+If you followed the instructions in _Creating certificates for a multi-cluster mesh_ then your certificates will reside in the directory specified in the previous example. If your certificates reside in a different directory, modify the syntax accordingly. 
+====
+
+. Log in to the second cluster by running the following command:
++
+[source,terminal]
+----
+$ $ oc login -u https://<second_cluster_api_server_url>
+----
+
+. Set up the environment variable that contains the context for the second cluster by running the following command:
++
+[source,terminal]
+----
+$ export CTX_CLUSTER1=$(oc config current-context)
+----
+
+. Create a namespace called `istio-system` by running the following command:
++
+[source,terminal]
+----
+$ oc get ns istio-system --context "${CTX_CLUSTER2}" || oc create namespace istio-system --context "${CTX_CLUSTER2}"
+----
+
+. Configure {istio} to use `network2` as the default network for the pods on the West cluster by running the following command:
++
+[source,terminal]
+----
+oc --context "${CTX_CLUSTER2}" label namespace istio-system topology.istio.io/network=network2
+----
+
+. Create the CA certificate secret for {istio} on the West cluster by running the following command:
++
+[source,terminal]
+----
+oc get secret -n istio-system --context "${CTX_CLUSTER2}" cacerts || oc create secret generic cacerts -n istio-system --context "${CTX_CLUSTER2}" \
+  --from-file=west/ca-cert.pem \
+  --from-file=west/ca-key.pem \
+  --from-file=west/root-cert.pem \
+  --from-file=west/cert-chain.pem
+----
++
+[NOTE]
+====
+If you followed the instructions in _Creating certificates for a multi-cluster mesh_ then your certificates will reside in the directory specified in the previous example. If the certificates reside in a different directory, modify the syntax accordingly. 
+====
+
+.Next steps
+
+Install the Operator. 
```

Comment:
Add name of the operator and that it has to be installed on both clusters.
```suggestion
Install the Service Mesh Operator on both clusters. 
```

Reply:
Also, maybe installing the operator could be the first step, so you get this sequence:
1. install operator on both clusters
2. create the certificates for both clusters
3. apply the certificates to both clusters
4. install istio on both clusters

This way, the "Next step" in this document would be more natural, since you need the certificates for Istio, not for the operator.